python/mcap/mcap0/writer.py

```diff
@@ -48,9 +48,9 @@ class Writer:
 
     def __init__(
         self,
-        output: Union[str, BytesIO, BufferedWriter],
+        output: Union[str, IO[Any]],
         chunk_size: int = 1024 * 1024,
-        compression: CompressionType = CompressionType.NONE,
+        compression: CompressionType = CompressionType.ZSTD,
```

Comment:
I think this change should definitely be made, maybe independent of the larger patch

Reply:
Yeah I made this in separate smaller PR. I'm going to close this PR in favor of another package that provides a separate ROS1 decoder class.